### PR TITLE
Draft release notes

### PR DESCRIPTION
**Pull request by csharpfritz**

### CODE REVIEW

File: NuGet.Docs/Release-Notes/NuGet-3.4.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    * Installing / Uninstalling packages that have install.ps1 / uninstall.ps1 scripts
+    * Loading projects that have an init.ps1 script (like EntityFramework)
+    * Publishing web content
+
+* __Workaround:__ Ensure that your Windows 10 install has the latest patches applied, expecially the January 2016 (KB 3124263) or a later update.  More details are available on [GitHub issue #1638](http://github.com/nuget/home/issues/1638)
+
+* __Issue:__ NuGet v2 protocol redirects are broken.
+Custom NuGet repositories that redirect requests to an alternative host do not honor the redirect request.
+* __Workaround:__  To work around this issue, configure the package repository URI in settings to point to the redirected server location.
+For more information, see [GitHub issue #387](http://github.com/nuget/home/issues/387).
```
Comment (xavierdecoster) [inline NuGet.Docs/Release-Notes/NuGet-3.4.md]: The referenced github issue does not seem to be related?

Comment (csharpfritz) [inline NuGet.Docs/Release-Notes/NuGet-3.4.md]: Rats.. that should be PR#387
